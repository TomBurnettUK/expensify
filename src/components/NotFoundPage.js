import React from 'react';
import { Link } from 'react-router-dom';

const NotFoundPage = () => (
  <div>
    <p>
      Not found! <Link to="/">Homepage</Link>
    </p>
  </div>
);

export default NotFoundPage;
