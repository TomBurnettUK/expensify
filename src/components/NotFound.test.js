import React from 'react';
import { shallow } from 'enzyme';
import NotFoundPage from './NotFoundPage';

it('should render correctly', () => {
  const wrapper = shallow(<NotFoundPage />);
  expect(wrapper).toMatchSnapshot();
});
