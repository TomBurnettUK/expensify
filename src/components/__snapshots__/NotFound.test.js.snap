// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly 1`] = `
<div>
  <p>
    Not found! 
    <Link
      replace={false}
      to="/"
    >
      Homepage
    </Link>
  </p>
</div>
`;
